src/pages/Home/components/FavoritesCitiesWeatherWidget/FavoritesCitiesWeatherWidget.tsx

```diff
@@ -1,16 +1,17 @@
 import { IconTypes } from 'enums/IconTypes';
+import { useLocalStorage } from 'hooks/useLocalStorage';
 import { CitiesWeatherWidget } from 'components/CitiesWeatherWidget';
 import { CityWeatherShortcut } from 'components/CityWeatherShortcut';
 import { FavoritesCitiesWeatherWidgetTestIds } from './FavoritesCitiesWeatherWidgetTestIds';
 
 const FavoritesCitiesWeatherWidget = () => {
-  const favoriteCities: string[] = [];
+  const [favoriteCities] = useLocalStorage<string[]>('favorites');
```

Comment:
**suggestion** I think this favorites list should be obtained from `context` instead of `local storage`

Reply:
Yes, you're right

Reply:
Done